Linked list with void pointers.c

```diff
@@ -37,37 +32,65 @@ void display_float(node *tail)
 {
     printf("%lf\n", *(double *)tail->data);
 }
-void display(void(*display_datatype)(node *tail), node *tail)
+void display_string(node *tail)
+{
+    printf("%c\n", *(char *)tail->data);

```

Comment:
is this char or string?

Reply:
that for char actually I will  change it